network/src/sync/blocks/aggro.rs

```diff
@@ -20,7 +20,6 @@ use crate::{Cache, Node, Payload, Peer, SyncBase, SyncInbound};
 use anyhow::*;
 use indexmap::IndexSet;
```

Comment:
I believe this is an unused import, would be good to remove as a fly-by.

Reply:
done in another PR